run_dir/design/assign_roles.html

```diff
@@ -0,0 +1,225 @@
+{% extends "base.html" %}
+
+<!--
+Template file: assign_roles.html
+URL: /assign_roles
+Title: displays all the users quthorized to use genomics-status
+Description: Shows a table with all genomics-status users with options to modify them.
+-->
+
+{% block stuff %}
+
+<h1 class="page_title"><span id="page_title">Assign Roles</span> <small>Create and modify users</small></h1>
+
+{% if user.role != 'admin'%}
+
+  <p>Sorry! Your user role does not allow you to view this page!</p>
+
+{% else %}
+  <div class="panel panel-default">
+    <div class="panel-body">
+      <button class="btn btn-primary" data-toggle="modal" type="button" data-target="#createUserModal">
+        <i class="glyphicon glyphicon-plus"></i> Create User
+      </button>
+      <button class="btn btn-primary" data-toggle="modal" type="button" data-target="#modifyUserModal">
+        <i class="glyphicon glyphicon-wrench"></i> Modify User
+      </button>
+    </div>
+  </div>
+  <table class="table table-striped table-bordered sortable" id="ur_table">
+    <thead>
+      <tr class="sticky">
+        <th class="sort" data-sort="user_name">User Name</th>
+        <th class="sort" data-sort="role">Role</th>
+      </tr>
+    </thead>
+    <tfoot>
+      <tr>
+        <th class="sort" data-sort="user_name">User Name</th>
+        <th class="sort" data-sort="role">Role</th>
+      </tr>
+    </tfoot>
+    <tbody class="list" id="user_table_body">
+    </tbody>
+  </table>
+
+  <div id="createUserModal" class="modal fade">
+    <div class="modal-dialog modal-md">
+      <div class="modal-content">
+        <div class="modal-header">
+          <button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
```

Comment:
I think "Cancel" is more appropriate instead of "Close" as a label for the button.

Reply:
Done!